lib/mrb/scripts/command_line/grndb.rb

```diff
@@ -603,8 +603,13 @@ def recover
           logger.log(:info, "Recovering database: <#{@database.path}>")
           if @force_truncate
             @database.each do |object|
```

Comment:
How about the following API?

```suggestion
            @database.each(use_temporary_open_space: true) do |object|
```